central/src/App.tsx

```diff
@@ -27,18 +27,20 @@ const theme = createTheme({
 
 function App() {
   const [alert, setAlert] = useState<Alert | null>(null);
-  const apiClients = new APIClients(Environment.Developing);
   const alertContext = {
     alert,
     setAlert,
   };
 
```

Comment:
useApiClient custom hook for initialization of AuthAPIClient first, and then other APIClients